src/etapi/attributes.js

```diff
@@ -20,7 +20,7 @@ function register(router) {
         'isInheritable': [v.notNull, v.isBoolean]
     };
 
-    eu.route(router, 'post' ,'/etapi/attributes', (req, res, next) => {
+    eu.route(router, 'post' ,'/etapi/attributes/:attributeId', (req, res, next) => {
```

Comment:
This change is rather PUT semantics, rather than POST.

Honestly it seems that the ETAPI OpenAPI spec is wrong, because it suggests the POST method requires the ID path param which isn't true. So I think we just need to fix the YAML file to better reflect the existing API.

Reply:
Agreed, and I tried to do that before by putting "parameters:" under each HTTP method, but it didn't work that way. For some reason I didn't think about moving POST into its own "/attributes" path rather than having it under "/attributes/{attributeId}", same for branches. But that worked like a charm and reduced this PR to only changes in the YAML file which is nice.